test/correctness/simd_op_check.cpp

```diff
@@ -1247,7 +1247,7 @@ struct Test {
             check(arm32 ? "vshll.u16" : "ushll", 4*w, u32(u16_1)*16);
             check(arm32 ? "vshll.u32" : "ushll", 2*w, u64(u32_1)*16);
 
-            // VSHR     I	-	Shift Right
+            // VSHR     I   -   Shift Right
```

Comment:
These changes are not part of the fix. They will simply corrupt the git blame history. Please undo these changes.